setup.py

```diff
@@ -55,7 +55,9 @@ def get_advanced_templates():
     entry_points={
         'console_scripts': [
             'release=release:main',
-            'ccm-deploy-test=test_util.test_installer_ccm:main',
+            # Remove this once CI is updated
+            'ccm-deploy-test=test_util.test_aws_vpc:main',
```

Comment:
agreed, soon we shall cleanse that. Just want to do it all at once rather than piecemeal